src/libsystemd-network/sd-dhcp-lease.c

```diff
@@ -1066,8 +1065,8 @@ int dhcp_lease_save(sd_dhcp_lease *lease, const char *lease_file) {
         _cleanup_fclose_ FILE *f = NULL;
         struct in_addr address;
         const struct in_addr *addresses;
-        const void *client_id, *data;
-        size_t client_id_len, data_len;
+        const void *data;
+        size_t data_len;
```

Comment:
btw, in the tpm2 code i started to switch cases like this (where you have some arbitrary blob pointer plus size) to use "struct iovec". There are a bunch of helpers to make working with such pointer+size pairs easy in iovec-util.h.

Might be somewthing to use here too, but doesn't matter

Reply:
Sounds nice. But I'd like to do that consistently. I will try that in a later PR.